SwaprHQ/swapr-dapp
# [SWA-134][IMPROV] - Fix CoW Trade type errors

## Fixes: [SWA-134](https://linear.app/swaprhq/issue/SWA-134/swapr-sdk-fix-cowtrade-type-errors)

# Description

* Point SDK PR for testing purposes

# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Swapr landing page
4) Connect your wallet
5) Any swap and bridge using CoW should be working, try using Gnosis Chain
